### PR TITLE
Formatted stats.py, added some documentation.

### PR DESCRIPTION
**Pull request by kurealnum**
Git is saying that I removed a line from `config_default.txt`. Not sure why, but it looks fine. Anyways, please let me know if you want to change anything with the documentation. It's pretty basic at the moment.

### CODE REVIEW

**Comment by morr0ne** [comment]
Thank you for your contribution! The documentation updates look great and are much appreciated. However, I have some concerns about the formatting changes. Currently, the project does not have a standard formatter set up, which might explain the discrepancies between the formatting in your PR and the formatting on other systems. It’s important to ensure consistency across different environments, so I suggest we establish a standard formatter as part of our CI process in a future update.

Regarding the changes to the configuration files, it would be best to remove them from this PR. Please rebase your branch to clean up those changes. In the future, be mindful to only stage (git add) the specific files or changes you intend to include in a PR.

Once these adjustments are made, I believe we can proceed with merging the documentation updates. We can address setting up a formatter and integrating it into our CI in a separate PR

File: README.md [hunk 1]
```diff
@@ -77,6 +77,43 @@ The sky's the limit when it comes to creativity! Whether you're aiming for a
 traditional system information display or something entirely unique, **pbfetch**
 empowers you to bring your vision to life.
 
+### Configuration Info
+
+Your configuration file must be placed in `$XDG_CONFIG_HOME/.config/pbfetch`, and the file itself must be called `config.txt`.
```
Comment (morr0ne) [inline README.md:82]: This wording could be improved. It’s a bit awkward as it currently stands, so it might benefit from some rephrasing

**Review by morr0ne — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -93,6 +130,16 @@ Creating your own custom configuration is a bit more involved than other fetch
...
 
+## 💻 Contributing
+
+Follow the [Django Code of Conduct](https://www.djangoproject.com/conduct/) when contributing (in short, just be a good hooman).
+
+When formatting Python code, use the [Black formatter](https://github.com/psf/black).
+
+### Making changes
+
+To make changes to this codebase, you first need to create a fork. Then, you can clone your fork and follow the steps under [installation via Rye](#install-via-rye) to get the project running. From there, you can commit and push any changes that you've made. Finally, you can create a pull request on Github.
```
[the remark this answers is not in the document]
Comment (kurealnum) [reply README.md:141]: Not sure what else to put here. I wanted to mention running the project via Rye, as I found that that was the easiest and quickest way to get it up and running. The rest is just boilerplate though. Would something like this sound better?

```
### Making changes
Install the project via [Rye](#install-via-rye). Then, follow Github's standard workflow for creating a pull request. 
```

File: README.md [hunk 3]
```diff
@@ -93,6 +130,16 @@ Creating your own custom configuration is a bit more involved than other fetch
 utilities, but the reward is complete control over your terminal’s output. You
 are not confined to someone else's design, **pbfetch** puts you in charge!
 
+## 💻 Contributing
+
+Follow the [Django Code of Conduct](https://www.djangoproject.com/conduct/) when contributing (in short, just be a good hooman).
```
[the remark this answers is not in the document]
Comment (kurealnum) [reply README.md:135]: In that case, I'll remove this from my fork and open an issue. Thank you!

**Comment by pennybelle** [comment]
Sorry about the unexpected close, I renamed the master branch to main which is why this closed automatically. Ive also made dev the default branch so please make new PRs on the dev branch instead of main (the replacement for master).

Since this closed automatically I don't see a way to reopen it, so if you'd like to just make a new PR on dev with the documentation changes we can continue from there.

**Comment by kurealnum** [comment]
That sounds fine! I'll open one soon.

**Pull request closed without merging**